launch/mcptam.launch

```diff
@@ -13,7 +13,7 @@
   <rosparam command="load" file="$(find mcptam)/groups/$(arg group_name).yaml" />
   
   <remap from="reset" to="mcptam/reset" />
-  <param name="image_transport" type="string" value="raw"/>
+  <param name="image_transport" type="string" value="compressed"/>
```

Comment:
Trying to avoid committing this to the master.  Unless we actually want to make it the default, which I'm fine with unless it slows performance.